FWCore/Catalog/src/FileLocator.cc

```diff
@@ -220,7 +228,9 @@ namespace edm {
 
     /*first of all do the lfn-to-pfn bit*/
     {
-      DOMNodeList* rules = doc->getElementsByTagName(_toDOMS("lfn-to-pfn"));
+      XMLCh* lfn_to_pfn = XMLString::transcode("lfn-to-pfn");
```

Comment:
Ok, I'll change it. Perhaps, I could reuse http://cmslxr.fnal.gov/source/DetectorDescription/Parser/interface/XercesString.h?v=CMSSW_8_1_X_2016-07-04-2300? (hmm.. just noticed, I missed release in destructor)